ippon_back/settings.py

```diff
@@ -26,7 +26,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = ['192.168.1.9', '192.168.1.11', 'ippon-backend.herokuapp.com', 'localhost']
+ALLOWED_HOSTS = ['192.168.1.9', '192.168.1.11','192.168.8.102', 'ippon-backend.herokuapp.com', 'localhost']
```

Comment:
This looks bad. I will have to get some mechanism to modify this list from environment.

Reply:
This file got committed by accident so it can be rejected. For development purposes, you could grab the IP in manage.py and append it to the list in settings. If you want I can write that today.

Reply:
And the load from env isn't the greatest idea because you may want to either host on localhost or on your internal network and changing it would be inconvenient.

Reply:
Heroku and Travis want sensitive data to be handed over through environment, so I assume that is the right way to do. I am not happy about requiring to modify any file in project to adjust for depolyment.

Only other passable option would be some kind of config file.

Reply:
The idea of modifying manage.py was only to make development easy. I see why you wouldn't like that idea, so we could just make a python script from which we would import the allowed hosts array, add it to .gitignore and have the setup script create it.